src/headphones2/postprocess/component_base.py

```diff
@@ -0,0 +1,13 @@
+import abc
+
+
+class PostProcessorComponentBase(object):
+    __metaclass__ = abc.ABCMeta
+
+    @staticmethod
```

Comment:
Using static method in this context is not recommended.
You will lose the ability to have a state for your processor, and it makes for a strange api.